src/Components/HeaderComponents/styles.tsx

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../styles/colors';
+
+export default StyleSheet.create({
+  allCasesBorder: {
+    paddingRight: 230,
+    marginTop: 14,
+    paddingBottom: 14,
+    borderBottomWidth: 1,
+    borderBottomColor: colors.darkGrey,
+  },
+  headerText: {
+    color: colors.midRed,
+    textAlign: 'center',
+    fontSize: 12,
+    fontStyle: 'normal',
+    fontWeight: '400',
+    lineHeight: 24,
+    letterSpacing: -0.264,
```

Comment:
I'm scared of this.

Reply:
I don't think it does anything should I take it out